### PR TITLE
Add a note about GitHub merge button

### PR DESCRIPTION
**Pull request by jbonofre**

### CODE REVIEW

**Comment by xumingming** [comment]
@asfgit merge

**Comment by asfgit** [comment]
MergeBot: Merge: PR 359 in state error with description Unexpected Error: 502 Server Error: Proxy Error for url: https://builds.apache.org/job/beam_PreCommit_Website_Merge/api/python?tree=builds%5Bnumber%2Curl%5D.. URL: http://mergebot-vm.apache.org/beam-website/359.

**Comment by asfgit** [comment]
Error: MergeBot encountered an unexpected error while processing this PR: 502 Server Error: Proxy Error for url: https://builds.apache.org/job/beam_PreCommit_Website_Merge/api/python?tree=builds%5Bnumber%2Curl%5D.

File: src/contribute/contribution-guide.md [hunk 1]
```diff
@@ -383,6 +383,14 @@ At this point, you want to ensure everything is right. Test it with `mvn verify`
 
 Done. You can delete the local `finish-pr-<pull-request-#>` branch if you like.
 
+It's also possible to use the "Merge Pull Request" button on GitHub, supporting three modes:
+
+* Merge
+* Squash & Merge
+* Rebase & Merge
+
+It's similar to do the merge by hand.
```
Comment (robertwb) [inline src/contribute/contribution-guide.md]: We should probably provide guidance on which one to use, or at least a pointer to the discussion thread.

**Comment by kennknowles** [comment]
I'll close just to tidy up pull requests because I think it is already live on the site. Feel free to reopen or propose edits. If you do, note that the committer guide, with all the merging info, is now a separate page.

**Pull request closed without merging**